packages/host/package.json

```diff
@@ -155,7 +155,7 @@
     "moment": "^2.29.4",
     "moment-locales-webpack-plugin": "^1.2.0",
     "monaco-editor": "^0.52.2",
-    "monaco-editor-webpack-plugin": "^7.0.1",
+    "monaco-editor-webpack-plugin": "^7.1.0",
```

Comment:
It looks like that when 7.1.1 will be released, we will be able to upgrade and remove the patch.